istio/istio-install/base/istio-noauth.yaml

```diff
@@ -14580,6 +14594,13 @@ metadata:
     app: istio-ingressgateway
     istio: ingressgateway
 spec:
+  selector:
+    matchLabels:
+      chart: gateways
+      heritage: Tiller
```

Comment:
nit can we get rid of the heritage label in the selector?

Reply:
not in this PR

Reply:
@jlewi Removal of Tiller has been done incrementally in this PR #378